lib/std/json.zig

```diff
@@ -1483,7 +1483,22 @@ fn parseInternal(comptime T: type, token: Token, tokens: *TokenStream, options:
                                         parseFree(field.field_type, @field(r, field.name), options);
                                     }
                                 }
-                                @field(r, field.name) = try parse(field.field_type, tokens, options);
+                                var parsed_value = try parse(field.field_type, tokens, options);
```

Comment:
should this be `const` rather than `var`?

Reply:
Yes it should